pep-0684.rst

```diff
@@ -10,7 +10,6 @@ Created: 08-Mar-2022
 Python-Version: 3.12
 Post-History: `08-Mar-2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/>`__,
               `29-Sep-2022 <https://discuss.python.org/t/pep-684-a-per-interpreter-gil/19583>`__,
-              `28-Oct-2022 <https://discuss.python.org/t/pep-684-a-per-interpreter-gil/19583/19>`__,
```

Comment:
This is fine, but it's unfortunate that the PEP will no longer point to the beginning of the third round of discussion.

Reply:
Mmm, so in light of your thoughtful explaination and reasoning presented on #3004 , I think I've come up with a workaround for the linter that will still detect cases where a specific message was unintentionally appended (e.g. due to scrolling), but still allow intentionally linking directly to a specific post if the author is convinced that is best for their PEP.

Specifically, we can have it not flag links to a specific Discourse message that end in a trailing slash, which isn't added to the links by default when scrolling or copying the link to a post, but can be added manually by authors who deliberately want to link to a particular message.

It's a little bit inconsistent since readers (and tools) likely generally expect these to be links to distinct threads, rather than different points in the same thread, but its unclear if its really _that_ much of a problem in practice currently, and if it is we can always go back and fix it later.

Reply:
Yeah, that works.

Reply:
Implemented :heavy_check_mark: